MediaBrowser.Controller/Plugins/IServerEntryPoint.cs

```diff
@@ -1,21 +1,15 @@
-using System;
 using System.Threading.Tasks;
 
 namespace MediaBrowser.Controller.Plugins
 {
     /// <summary>
     /// Interface IServerEntryPoint
     /// </summary>
-    public interface IServerEntryPoint : IDisposable
+    public interface IServerEntryPoint
```

Comment:
Maybe this should be renamed to trigger a compilation error in plugins instead of breaking them be accident. Perhaps `IStartupTask` or `IServerStartupTask`?